README.adoc

```diff
@@ -298,34 +298,4 @@ So you can use it through:
 @Inject
 @ConfigProperty(name = "app.service.url")
 private String url;
-----
-
-== Swagger Integration
```

Comment:
does it need to be dropped? was a key feature, no issue to move to openapi if swagger is no more desired but having a swagger-ui compat was important

Reply:
I don't know how to deal with it.
I removed it because it relied on [swagger-play2 library] that is archived and not released under scala 2.13.
Since play is no more compatible with scala 2.12, it's supposed to be binary imcompatible.
That's why I preferred dropping it.

Reply:
Ok but then we need to document it (not just drop the section but mark it as no more supported to be aligned on the upstream stack maybe) and move to next major